test/worldwide/region_test.rb

```diff
@@ -263,6 +263,27 @@ class RegionTest < ActiveSupport::TestCase
       end
     end
 
+    test "associated_continent returns the expected continent" do
+      [
+        # Canada
+        [:ca, "North America"],
+        # Ontario is a province of Canada, which is in North America
+        ["CA-ON", "North America"],
+        # Tokyo is a province of Japan, which is in Asia
+        ["JP-13", "Asia"],
+        # North America itself
+        ["003", "North America"],
+        # Peurto Rico is a territory of the US, which is in North America too
+        ["US-PR", "North America"],
```

Comment:
This example demonstrates that not every "first parent" leads to a continent.  The Caribbean (029)'s first parent is Latin America (419), and the second is North America (003), a continent 👍